README.md

```diff
@@ -10,13 +10,16 @@
 - `with-node`, `85-with-node`, `85-with-node-12` [(Dockerfile)](https://github.com/Zenika/alpine-chrome/blob/master/with-node/Dockerfile)
 - `with-puppeteer`, `85-with-puppeteer` [(Dockerfile)](https://github.com/Zenika/alpine-chrome/blob/master/with-puppeteer/Dockerfile)
 - `with-playwright`, `85-with-playwright` [(Dockerfile)](https://github.com/Zenika/alpine-chrome/blob/master/with-playwright/Dockerfile)
+- `with-selenoid`, `85-with-selenoid` [(Dockerfile)](https://github.com/Zenika/alpine-chrome/blob/master/with-selenoid/Dockerfile)
```

Comment:
Do you want to add these tags: `85-with-selenoid-1.10` and `85-with-chromedriver-85` ?

Reply:
I'm not sure they needed much, since, for the `with-chromedriver`, the version of ChromeDriver usually should be paired with the Chrome version.

Regarding the `with-selenoid` this is the better question. We could pin the version, but using `latest-stable` should suffice for a start.